core/src/com/unciv/logic/map/mapgenerator/mapregions/LuxuryResourcePlacementLogic.kt

```diff
@@ -239,7 +239,8 @@ object LuxuryResourcePlacementLogic {
         for (region in regions) {
             val tilesToCheck = tileMap[region.startPosition!!].getTilesInDistanceRange(1..2)
             val candidateLuxuries = randomLuxuries.shuffled().toMutableList()
-            if (tileMap.mapParameters.mapResources != MapResources.strategicBalance)
+            if (tileMap.mapParameters.mapResources != MapResources.strategicBalance &&
```

Comment:
mapResources can also be strategic balance/legendary start? Do we want to support both types, or are we deprecating the old one? If the first, I don't understand why. If the second, we should add deprecation warnings on those mapresource vals, for warning.

Reply:
I'd say the second